docs/cli/commands/botkube_uninstall.md

```diff
@@ -30,7 +30,7 @@ botkube uninstall --release-name botkube-dev
   -y, --auto-approve          Skips interactive approval for deletion.
       --cascade string        Must be "background", "orphan", or "foreground". Selects the deletion cascading strategy for the dependents. Defaults to background. (default "background")
       --description string    add a custom description
-      --dry-run               simulate a uninstall
+      --dry-run               simulate an uninstallation
```

Comment:
This will be overriden next time as the docs are generated on `botkube` repo. But worry not, I've already created a PR with fix 👍 https://github.com/kubeshop/botkube/pull/1234